examples/preprocessing/plot_map_data_to_normal.py

```diff
@@ -53,7 +53,8 @@
 rng = np.random.RandomState(304)
 bc = PowerTransformer(method='box-cox')
 yj = PowerTransformer(method='yeo-johnson')
-qt = QuantileTransformer(output_distribution='normal', random_state=rng)
+qt = QuantileTransformer(n_quantiles=500, output_distribution='normal',
```

Comment:
I would add a comment explaining that n_quantiles is set to the training set size to avoid a warning.

Reply:
"Avoiding the warning" is doing the correct thing in this case. In this case, `n_quantiles=500` is the number of samples in `X_train` (because `test_size=0.5` and `N_SAMPLES=1000`).

Reply:
I agree but I would make this explicit so that someone reading the example or changing the code will know why.
Unless tests will be failing if a warning is raised?

Reply:
@KenitoInc I am sorry if I was not clear but what I meant was more something like "n_quantiles is set to the training set size rather than the default value to avoid a warning being raised by this example"

Reply:
Thanks @albertcthomas I have updated the PR